### PR TITLE
Linking to the EditImageActivity from picLeaf

### PR DESCRIPTION
**Pull request by Subhankar29**
Fixes issue #461

Changes: Now choosing the edit option from the gallery will direct to the EditImageActivity class. 

Screenshots for the change: 
screenshot_2017-05-21-08-16-11-330

### CODE REVIEW

File: imageeditlibrary/build.gradle [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 24
+    buildToolsVersion '25.0.0'
```
Comment (pa1pal) [inline imageeditlibrary/build.gradle]: change it to `25.0.3`

File: imageeditlibrary/build.gradle [hunk 2]
```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 24
```
Comment (pa1pal) [inline imageeditlibrary/build.gradle]: change it to `25` to fix the fail checks

**Comment by pa1pal** [comment]
Crash log:
`Unable to decode stream: java.io.FileNotFoundException: file:/storage/emulated/0/DCIM/Camera/IMG_20170518_103609.jpg (No such file or directory)
05-24 13:36:52.814 5410-5860/vn.mbm.phimp.me E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #4
                                                               Process: vn.mbm.phimp.me, PID: 5410
                                                               java.lang.RuntimeException: An error occurred while executing doInBackground()
                                                                   at android.os.AsyncTask$3.done(AsyncTask.java:325)
                                                                   at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
                                                                   at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
                                                                   at java.util.concurrent.FutureTask.run(FutureTask.java:242)
                                                                   at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:243)
                                                                   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
                                                                   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
                                                                   at java.lang.Thread.run(Thread.java:761)
                                                                Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.graphics.Bitmap.getWidth()' on a null object reference
                                                                   at android.graphics.Bitmap.createBitmap(Bitmap.java:681)
                                                                   at com.xinlan.imageeditlibrary.editimage.task.StickerTask.doInBackground(StickerTask.java:47)
                                                                   at com.xinlan.imageeditlibrary.editimage.task.StickerTask.doInBackground(StickerTask.java:21)
                                                                   at android.os.AsyncTask$2.call(AsyncTask.java:305)
                                                                   at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                                   at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:243) 
                                                                   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133) 
                                                                   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607) 
                                                                   at java.lang.Thread.run(Thread.java:761) 
05-24 13:36:52.841 6239-29936/system_process E/ActivityManager: Sending non-protected broadcast projekt.substratum.APP_CRASHED from system 6239:system/1000 pkg android
                                                                java.lang.Throwable
                                                                    at com.android.server.am.ActivityManagerService.checkBroadcastFromSystem(ActivityManagerService.java:18102)
                                                                    at com.android.server.am.ActivityManagerService.broadcastIntentLocked(ActivityManagerService.java:18691)
                                                                    at com.android.server.am.ActivityManagerService.broadcastIntent(ActivityManagerService.java:18782)
                                                                    at android.app.ContextImpl.sendBroadcast(ContextImpl.java:881)
                                                                    at com.android.server.am.AppErrors.crashApplicationInner(AppErrors.java:374)
                                                                    at com.android.server.am.AppErrors.crashApplication(AppErrors.java:308)
                                                                    at com.android.server.am.ActivityManagerService.handleApplicationCrashInner(ActivityManagerService.java:13682)
                                                                    at com.android.server.am.ActivityManagerService.handleApplicationCrash(ActivityManagerService.java:13664)
                                                                    at android.app.ActivityManagerNative.onTransact(ActivityManagerNative.java:1669)
                                                                    at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2971)
                                                                    at android.os.Binder.execTransact(Binder.java:565)
05-24 13:36:53.552 32048-6128/com.google.android.gms.persistent E/AppUsageEventWatcher: wrong time range: [1495648683725, 1495613213548)
05-24 13:37:05.824 3250-3250/? E/NEW_BHD: Open /sys/class/power_supply/gb_battery`

**Comment by Subhankar29** [comment]
Yes I am working on that.

**Comment by mariobehling** [comment]
Please get PRs peer-reviewed. At least two people and to establish this process it would be preferable to have three people reviewing at the beginning.

Please peer-review now! Ask other devs and assign them by using "@" in your PRs.

**Comment by Subhankar29** [comment]
@mariobehling Ok. I will do that.

**Comment by Subhankar29** [comment]
@vinaysajjanapu @mohitmanuja @pa1pal Please review the PR.

**Pull request merged**